src/metabase/search/appdb/scoring.clj

```diff
@@ -51,8 +53,10 @@
       [:- ceiling
        [:/
         ;; Use seconds for granularity in the fraction.
-        ;; TODO will probably need to specialize this based on (mdb/db-type)
-        [[:raw "EXTRACT(epoch FROM (" [:- to-column from-column] [:raw "))"]]]
+        (case (mdb/db-type)
+          :mysql
+          [[:raw "TIMESTAMPDIFF(second, " (sql/format-expr from-column) ", " (sql/format-expr to-column) ")"]]
```

Comment:
I'm not sure how to pass `from-column` here. If I do `format-expr`, it becomes a string and is put into a parameter. If I don't do that, it errors out with `unknown escape sequence {:select [[[:max :recent_views.timestamp]...`. 🙄

Reply:
Not sure I understand, but what about `(keyword (sql/format-expr ...))`? I don't understand why having a naked expression works for the other drivers but not MySQL, can you sure the full error?